spec/system/device_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe "user", type: :feature do
+  before :each do
+    User.create(email: 'user@example.com', password: 'password')
+  end
+
+  it "creates new user" do
+    visit new_user_registration_path
+
+    fill_in "user_first_name", with: "Harry"
+    fill_in "user_last_name", with: "Potter"
+    fill_in "user_email", with: "hp2@gmail.com"
+    fill_in "user_password", with: "Admin@123"
+    fill_in "user_password_confirmation", with: "Admin@123"
+
+
+    click_button 'Sign up'
+    expect(page).to have_content 'Sign Out'
```

Comment:
How about checking the page is redirected to the dashboard instead?

Reply:
resolved in https://github.com/owaisrazaeng1/nimble-test/commit/7aa833bd51593f376ce1c36a37c31e7b314e627f